flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -64,6 +64,7 @@
 import org.apache.flink.streaming.runtime.streamstatus.StreamStatusMaintainer;
```

Comment:
Commit message should be a bit more expressive, for example: Drain mailbox while shutting down StreamTask